notebooks/214-grammar-correction/214-grammar-correction.ipynb

```diff
@@ -62,12 +62,12 @@
   },
```

Comment:
<p>!pip install -q 'openvino-dev&gt;=2023.0.0'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p><code>openvino</code>,<code>onnx</code> and <code>onnxruntime</code> packages are installed with <code>optimum-intel[openvino]</code> . So this string becomes <code>!pip install "optimum-intel[openvino] @ git+https://github.com/huggingface/optimum-intel.git" gradio</code>  <span class="ql-cursor">﻿</span></p>

Reply:
@itrushkin  @igor-davidyuk  it was made intentionally, optimum pin specific version of openvino and its update schedule is not aligned with us, so it may lead to misalignment and accidental reinstalling openvino during optimum install (with upgrading or downgrading version), that can be unexpected, so installing separately is better choise

Reply:
Added installation of OpenVINO pinned to `2023.0.0`